keras_hub/src/tokenizers/byte_pair_tokenizer_test.py

```diff
@@ -15,7 +15,7 @@
 )
 
 
-@pytest.mark.large
+# @pytest.mark.large
```

Comment:
I think it'd be better to remove unused codes so if we don't want to mark this as large anymore, it'd be better to remove the line.

Reply:
ok

Reply:
removed